src/common/mailing.ts

```diff
@@ -114,7 +114,7 @@ export const syncSubscription = async function (
         );
       if (!digest) {
         await manager.getRepository(UserPersonalizedDigest).delete({
-          id: customer.id,
+          userId: customer.id,
```

Comment:
damn did we delete more than we should have?

Reply:
id doesn't exist :)  (so should be ok)

Reply:
Nah, just got error. That row doesn't exist anyways. Though, strange that linter didn't catch that the table doesn't have `id` column
 
> Property "id" was not found in "UserPersonalizedDigest". Make sure your query is correct.

Reply:
That would've been bad stupid from their side :triggered:

Reply:
I have to speed up that `strict` mode update.. it would have caught this

Reply:
Not sure... this is the type definition
<img width="712" alt="image" src="https://github.com/dailydotdev/daily-api/assets/1681525/2d1976a6-9c9d-4d50-8376-1dfa3f2afa38">

Reply:
[strict](https://www.typescriptlang.org/tsconfig/#strict) mode in typescript yes

Reply:
But even then it would not since their internal `ObjectId` always has `id` property 🤦

Reply:
Yeah 😿